OpenRA.Game/Network/OrderManager.cs

```diff
@@ -52,7 +52,7 @@ public sealed class OrderManager : IDisposable
 
 		bool disposed;
 
-		void OutOfSync(int frame)
+		public void OutOfSync(int frame)
```

Comment:
Yes, this was a left-over. I trigger an `OutOfSync` exception after 500 frames to test this.